addons/website/static/src/services/website_service.js

```diff
@@ -105,6 +110,20 @@ export const websiteService = {
             get currentWebsiteId(){
                 return currentWebsiteId;
             },
+            addWebsiteId(id) {
+                if (!currentWebsiteIdList.length){
+                    currentWebsiteId = id;
+                }
+                currentWebsiteIdList.push(id);
+            },
+            removeWebsiteId(){
+                currentWebsiteIdList.shift();
+                if (currentWebsiteIdList.length) {
+                    currentWebsiteId = currentWebsiteIdList[0];
+                } else {
+                    currentWebsiteId = null;
+                }
+            },
```

Comment:
don t expose it in the api. Could move it in the service scope ?
Just move the code before the return :)